src/dev/tsconfig.dev.json

```diff
@@ -6,6 +6,9 @@
     "moduleResolution": "node",
     "target": "es5"
   },
-
+  "angularCompilerOptions": {
```

Comment:
My ask is to move this to the root `tsconfig.json` instead so its the default project wide.

Reply:
I can try but it was not my intend to change the build settings for the whole project - just to fix #3932